### ISSUE TITLE
[Accessibility][Screen Reader-AzureLabServices-Users] On the Azure lab service page, Screen Reader does not announce the sorting information on the "name" column header of the table.

### ISSUE THREAD

**Issue by Saurabh19Aug** [OfficeDev/office-ui-fabric-react#12200]
**User Experience:** 
If, the Screen Reader does not announce the sorting information of the column header of the table, then the user who depends upon Screen Reader for navigation might get affected hence failed to get the sorting information of the table column header.

**Note:** User credentials should NOT be included in the bug.

**Pre Request:**
Turn on Narrator via using Ctrl+Window+Enter key combination. 

**Repro-Steps:**
1. Launch #https://insiders2.labs.azure.com and login via valid credentials.
2. Navigate to Azure Lab services window and select "Any already created lab" tile.
3. Navigate to left nav bar and select "Users" list item.
4. Navigate all the controls of "Users" window.
5. Navigate with the Narrator and reach "Name" column header of the table.
6. Verify, after pressing spacebar/enter key does the Screen Reader does announce the sorting information of the table or not.

**Actual Result:**
On the Azure lab service page, via pressing spacebar/ enter key Screen Reader does not announce the sorting information on the Name column header of the table. Instead Screen Reader announce space only.

**Expected Result:**
On the Azure lab service page, via pressing spacebar/ enter key Screen Reader should announce the sorting information on the Name column header of the table i.e name button row 1of 3, column 2 of 6 ascending/descending. 

**MAS Reference:**
MAS 1.3.1 - Info and Relationships

**Note:** 1. This issue also exist with Chrome +JAWS.
2. Screen reader pass the table cell value information in one-go.
3. Same issue exist throughout the application.

**Reference Links**
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries
[MAS1.3.1-Screen Reader does not announce the sorting key information of the table.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/4291655/MAS1.3.1-Screen.Reader.does.not.announce.the.sorting.key.information.of.the.table.zip)

**Comment by aneeshack4** [comment]
@Saurabh19Aug You must provide an isolated repro on the Fabric component itself - it is not actionable for us to be directed to partner sites. Please provide a codepen with steps to repro this on the Fabric component.

**Comment by Saurabh19Aug** [comment]
We have logged this bug in incorrect format. So closing the bug for now . We will log this bug later.
